docs/en/sql-reference/operators/exists.md

```diff
@@ -0,0 +1,31 @@
+# EXISTS {#exists-operator}
+
+The `EXISTS` operator tests how many records are in the result of a subquery. If it is empty, then the operator returns `0`. Otherwise it returns `1`.
```

Comment:
```suggestion
The `EXISTS` operator checks how many records are in the result of a subquery. If it is empty, then the operator returns `0`. Otherwise, it returns `1`.
```